### PR TITLE
TSK-1283: replaced usernames with - instead of _

### PR DESCRIPTION
**Pull request by holgerhagen**

### CODE REVIEW

**Comment by mustaphazorgati** [comment]
Hi @holgerhagen 
Can you please verify that all usages of the @WithAccessId Annotation are correct?
Here is an extract:
```  @WithAccessId(user = "admin")
  @WithAccessId(user = "admin", groups = "admin")
  @WithAccessId(user = "admin", groups = "businessadmin")
  @WithAccessId(user = "admin", groups = "group-1")
  @WithAccessId(user = "admin", groups = "group-2")
  @WithAccessId(user = "afterach2")
  @WithAccessId(user = "afterall")
  @WithAccessId(user = "afterall2")
  @WithAccessId(user = "aftereach")
  @WithAccessId(user = "beforeall")
  @WithAccessId(user = "beforeall2")
  @WithAccessId(user = "beforeeach")
  @WithAccessId(user = "beforeeach2")
  @WithAccessId(user = "businessadmin")
    @WithAccessId(user = "constructor")
  @WithAccessId(user = "dummy")
  @WithAccessId(user = "dummy", groups = "admin")
  @WithAccessId(user = "dummy", groups = "businessadmin")
  @WithAccessId(user = "elena")
  @WithAccessId(user = "Elena", groups = "businessadmin")
  @WithAccessId(user = "henry", groups = "undefinedgroup")
  @WithAccessId(user = INSIDE_DYNAMIC_TEST_USER)
  @WithAccessId(user = "max")
  @WithAccessId(user = "monitor")
  @WithAccessId(user = "monitor", groups = "admin")
  @WithAccessId(user = "taskadmin")
  @WithAccessId(user = "taskadmin", groups = "group-1")
  @WithAccessId(user = "teamlead-1")
  @WithAccessId(user = "teamlead-1", groups = "admin")
  @WithAccessId(user = "teamlead-1", groups = "businessadmin")
  @WithAccessId(user = "teamlead-1", groups = "group-1")
  @WithAccessId(user = "teamlead-1", groups = "group-1_1")
  @WithAccessId(user = "teamlead-1", groups = "teamlead-1")
  @WithAccessId(user = "TEAMLEAD-2", groups = "businessadmin")
  @WithAccessId(user = "testfactory")
  @WithAccessId(user = "testfactory1")
  @WithAccessId(user = "testfactory2")
  @WithAccessId(user = "testtemplate")
  @WithAccessId(user = "testtemplate1")
  @WithAccessId(user = "testtemplate1", groups = "abc")
  @WithAccessId(user = "testtemplate2")
  @WithAccessId(user = "testtemplate3")
  @WithAccessId(user = "unknown")
  @WithAccessId(user = "unknown", groups = "admin")
  @WithAccessId(user = "unknown", groups = "businessadmin")
  @WithAccessId(user = "user")
  @WithAccessId(user = "user-1-1")
  @WithAccessId(user = "user-1-1") // , groupNames = {"businessadmin"})
  @WithAccessId(user = "user-1-1", groups = "admin")
  @WithAccessId(user = "user-1-1", groups = "businessadmin")
  @WithAccessId(user = "user-1-1", groups = "group-1")
  @WithAccessId(user = "user-1-1", groups = "group-2")
  @WithAccessId(user = "user-1-1", groups = "taskadmin")
  @WithAccessId(user = "user-1-1", groups = "teamlead-1")
  @WithAccessId(user = "user-1-2")
  @WithAccessId(user = "user-1-2", groups = "businessadmin")
  @WithAccessId(user = "user-1-2", groups = "group-1")
  @WithAccessId(user = "user-1-2", groups = "group-2")
  @WithAccessId(user = "user-1-4", groups = "group-1")
  @WithAccessId(user = "user2")
  @WithAccessId(user = "user-2-2", groups = "businessadmin")
  @WithAccessId(user = "user-3-2", groups = "group-2")
  @WithAccessId(user = "User", groups = "businessadmin")
```
Unique user/groups: 
```"abc"
"admin"
"afterall"
"aftereach"
"beforeall"
"beforeeach"
"businessadmin"
"constructor"
"dummy"
"elena"
"Elena"
"henry"
"max"
"monitor"
"taskadmin"
"testfactory"
"testtemplate"
"undefinedgroup"
"unknown"
"user"
"User"
```
These are the users defined in taskana.property files (filtered by unique lines). Can you please make sure that all `taskana.roles.XXX` properties are line-identical? thanks!
```
taskana.roles.Admin=admin
taskana.roles.Admin=name=konrad,Organisation=novatec | admin
taskana.roles.Admin=name=konrad,Organisation=novatec|admin
taskana.roles.Admin=name=konrad,Organisation=novatec|admin|demo
taskana.roles.businessadmin=businessadmin | cn=business-admins,cn=groups,OU=Test,O=TASKANA
taskana.roles.businessadmin=max | Moritz | businessadmin
taskana.roles.businessadmin=max|Moritz|businessadmin
taskana.roles.businessadmin=max|Moritz|businessadmin| cn=business-admins,cn=groups,OU=Test,O=TASKANA
taskana.roles.businessadmin=max|Moritz|businessadmin|demo
taskana.roles.monitor=cn=monitor-users,cn=groups,OU=Test,O=TASKANA
taskana.roles.monitor=john | teamlead-2 | monitor
taskana.roles.monitor=john | teamlead_2 | monitor
taskana.roles.monitor=john|teamlead-2 | monitor
taskana.roles.monitor=john|teamlead_2 | monitor
taskana.roles.taskadmin=peter | taskadmin
taskana.roles.user = group1 | group2|teamlead-1 | teamlead-2 |user-1-1| user-1-1| user-1-2| user-2-1| user-2-2| max|elena|simone
taskana.roles.user=group1 | group2|teamlead-1 | teamlead-2 |user-1-1| user-1-1| user-1-2| user-2-1| user-2-2| max|elena|simone
taskana.roles.user = group1 | group2 | teamlead-1 | teamlead-2 | user-1-1 | user-1-1 | user-1-2 | user-2-1 | user-2-2 | max | elena | simone
taskana.roles.user = group1 | group2|teamlead-1 |teamlead-2 |user-1-1| user-1-1| user-1-2| user-2-1| user-2-2| max|elena|simone|demo
taskana.roles.user = group1 | group2|teamlead-1 |teamlead-2 |user-1-1| user-1-2| user-2-1| user-2-2| max|elena|simone
taskana.roles.user = teamlead-1 |teamlead-2 |user-1-1| user-1-2| user-2-1| user-2-2
```

These are our users according to the ldif files: 
```
dn: uid=admin,cn=users,OU=Test,O=TASKANA
dn: uid=businessadmin,cn=users,OU=Test,O=TASKANA
dn: uid=teamlead-1,cn=users,OU=Test,O=TASKANA
dn: uid=teamlead-2,cn=users,OU=Test,O=TASKANA
dn: uid=user-1-1,cn=users,OU=Test,O=TASKANA
dn: uid=user-1-2,cn=users,OU=Test,O=TASKANA
dn: uid=user-2-1,cn=users,OU=Test,O=TASKANA
dn: uid=user-2-2,cn=users,OU=Test,O=TASKANA
uid: admin
uid: businessadmin
uid: elena
uid: max
uid: simone
uid: teamlead-1
uid: teamlead-2
uid: user001
uid: user002
uid: user003
uid: user004
uid: user005
uid: user006
uid: user007
uid: user008
uid: user009
uid: user010
uid: user011
uid: user012
uid: user013
uid: user014
uid: user015
uid: user016
uid: user017
uid: user018
uid: user019
uid: user020
uid: user021
uid: user022
uid: user023
uid: user024
uid: user025
uid: user026
uid: user027
uid: user028
uid: user029
uid: user030
uid: user031
uid: user032
uid: user033
uid: user034
uid: user035
uid: user036
uid: user037
uid: user038
uid: user039
uid: user040
uid: user041
uid: user042
uid: user043
uid: user044
uid: user045
uid: user046
uid: user047
uid: user048
uid: user049
uid: user050
uid: user051
uid: user052
uid: user053
uid: user054
uid: user055
uid: user056
uid: user057
uid: user058
uid: user059
uid: user060
uid: user061
uid: user062
uid: user063
uid: user064
uid: user065
uid: user066
uid: user067
uid: user068
uid: user069
uid: user070
uid: user071
uid: user072
uid: user073
uid: user074
uid: user075
uid: user076
uid: user077
uid: user078
uid: user079
uid: user080
uid: user081
uid: user082
uid: user083
uid: user084
uid: user085
uid: user086
uid: user087
uid: user088
uid: user089
uid: user090
uid: user091
uid: user092
uid: user093
uid: user094
uid: user095
uid: user096
uid: user097
uid: user098
uid: user099
uid: user100
uid: user101
uid: user102
uid: user103
uid: user104
uid: user105
uid: user106
uid: user107
uid: user108
uid: user-1-1
uid: user110
uid: user111
uid: user112
uid: user113
uid: user114
uid: user115
uid: user116
uid: user117
uid: user118
uid: user119
uid: user-1-2
uid: user120
uid: user121
uid: user122
uid: user123
uid: user124
uid: user125
uid: user126
uid: user127
uid: user128
uid: user129
uid: user130
uid: user131
uid: user132
uid: user133
uid: user134
uid: user135
uid: user136
uid: user137
uid: user138
uid: user139
uid: user140
uid: user141
uid: user142
uid: user143
uid: user144
uid: user145
uid: user146
uid: user147
uid: user148
uid: user149
uid: user150
uid: user151
uid: user152
uid: user153
uid: user154
uid: user155
uid: user156
uid: user157
uid: user158
uid: user159
uid: user160
uid: user161
uid: user162
uid: user163
uid: user164
uid: user165
uid: user166
uid: user167
uid: user168
uid: user169
uid: user170
uid: user171
uid: user172
uid: user173
uid: user174
uid: user175
uid: user176
uid: user177
uid: user178
uid: user179
uid: user180
uid: user181
uid: user182
uid: user183
uid: user184
uid: user185
uid: user186
uid: user187
uid: user189
uid: user190
uid: user191
uid: user192
uid: user193
uid: user194
uid: user195
uid: user196
uid: user197
uid: user198
uid: user199
uid: user200
uid: user202
uid: user203
uid: user204
uid: user205
uid: user206
uid: user207
uid: user208
uid: user209
uid: user-2-1
uid: user210
uid: user211
uid: user212
uid: user213
uid: user214
uid: user215
uid: user216
uid: user217
uid: user218
uid: user219
uid: user-2-2
uid: user220
uid: user221
uid: user222
uid: user223
uid: user224
uid: user225
uid: user226
uid: user227
uid: user228
uid: user229
uid: user230
uid: user231
uid: user232
uid: user233
uid: user234
uid: user235
uid: user236
uid: user238
uid: user239
uid: user240
uid: user241
uid: user242
uid: user243
uid: user244
uid: user245
uid: user246
uid: user247
uid: user248
uid: user249
uid: user250
uid: user251
uid: user252
uid: user253
uid: user254
uid: user255
uid: user256
uid: user257
uid: user258
uid: user261
uid: user262
uid: user263
uid: user264
uid: user265
uid: user266
uid: user267
uniquemember: uid=businessadmin,cn=users,OU=Test,O=TASKANA
uniquemember: uid=teamlead-1,cn=users,OU=Test,O=TASKANA
uniquemember: uid=teamlead-2,cn=users,OU=Test,O=TASKANA
uniquemember: uid=user-1-1,cn=users,OU=Test,O=TASKANA
uniquemember: uid=user-1-2,cn=users,OU=Test,O=TASKANA
uniquemember: uid=user-2-1,cn=users,OU=Test,O=TASKANA
uniquemember: uid=user-2-2,cn=users,OU=Test,O=TASKANA
```

For me these three sources still don't match.

File: lib/taskana-core/src/test/resources/taskana.properties [hunk 1]
```diff
@@ -1,8 +1,8 @@
-taskana.roles.user=group1 | group2|teamlead-1 | teamlead-2 |user-1-1| user-1-1| user-1-2| user-2-1| user-2-2| max|elena|simone
-taskana.roles.Admin=name=konrad,Organisation=novatec|admin
-taskana.roles.businessadmin=max|Moritz|businessadmin
-taskana.roles.monitor=john|teamlead-2 | monitor
-taskana.roles.taskadmin=peter | taskadmin
+taskana.roles.user= teamlead-1 | teamlead-2 |user-1-1| user-1-1| user-1-2| user-2-1| user-2-2
+taskana.roles.Admin=admin |uid=admin,cn=users,OU=Test,O=TASKANA
+taskana.roles.businessadmin=businessadmin|TEAMLEAD-2
```
Comment (mustaphazorgati) [inline lib/taskana-core/src/test/resources/taskana.properties:3]: Is there a reason that teamlead-2 is in caps?

File: lib/taskana-data/src/main/resources/sql/monitor-data/monitor-sample-data.sql [hunk 2]
```diff
@@ -29,53 +29,53 @@ INSERT INTO ATTACHMENT VALUES('ATT:000000000000000000000000000000000010', 'TKI:0
 INSERT INTO ATTACHMENT VALUES('ATT:000000000000000000000000000000000011', 'TKI:000000000000000000000000000000000045', RELATIVE_DATE(0) , RELATIVE_DATE(0) , 'L99000'          ,'CLI:000000000000000000000000000000000009', ''         , ''        , ''          , ''      , ''       , ''     , RELATIVE_DATE(0) , null              );
```
Comment (mustaphazorgati) [inline lib/taskana-data/src/main/resources/sql/monitor-data/monitor-sample-data.sql:29]: I trust you on this file ;)

File: rest/taskana-rest-spring-example-common/src/test/resources/taskana.properties [hunk 3]
```diff
@@ -1,8 +1,8 @@
-taskana.roles.user = group1 | group2|teamlead-1 |teamlead-2 |user-1-1| user-1-2| user-2-1| user-2-2| max|elena|simone
-taskana.roles.Admin=name=konrad,Organisation=novatec|admin
-taskana.roles.businessadmin=max|Moritz|businessadmin| cn=business-admins,cn=groups,OU=Test,O=TASKANA
-taskana.roles.monitor=john|teamlead-2 | monitor
-taskana.roles.taskadmin=peter | taskadmin
+taskana.roles.user = teamlead-1 |teamlead-2 |user-1-1| user-1-2| user-2-1| user-2-2
+taskana.roles.Admin=admin
```
Comment (mustaphazorgati) [inline rest/taskana-rest-spring-example-common/src/test/resources/taskana.properties:2]: can you please make the 'Admin' lowercase?

File: rest/taskana-rest-spring-example-wildfly/src/main/resources/taskana.properties [hunk 4]
```diff
@@ -1,8 +1,9 @@
-taskana.roles.user = group1 | group2|teamlead-1 |teamlead-2 |user-1-1| user-1-1| user-1-2| user-2-1| user-2-2| max|elena|simone|demo
-taskana.roles.Admin=name=konrad,Organisation=novatec|admin|demo
-taskana.roles.businessadmin=max|Moritz|businessadmin|demo
-taskana.roles.monitor=john|teamlead_2 | monitor
-taskana.roles.taskadmin=peter | taskadmin
+taskana.roles.user = teamlead-1 |teamlead-2 |user-1-1| user-1-1| user-1-2| user-2-1| user-2-2
+taskana.roles.Admin=admin
```
Comment (mustaphazorgati) [inline rest/taskana-rest-spring-example-wildfly/src/main/resources/taskana.properties:2]: Please make 'Admin' lowercase

File: lib/taskana-core/src/test/java/acceptance/classification/UpdateClassificationAccTest.java [hunk 5]
```diff
@@ -80,28 +80,9 @@ void testUpdateClassification()
...
-    classification.setCustom6("newCustom6");
-    classification.setCustom7("newCustom7");
-    classification.setCustom8("newCustom8");
-    classification.setDescription("newDescription");
-    classification.setIsValidInDomain(false);
-    classification.setName(newName);
-    classification.setParentId("T2000");
-    classification.setPriority(1000);
-    classification.setServiceLevel("P2DT3H4M");
-
```
[the remark this answers is not in the document]
Comment (holgerhagen) [reply lib/taskana-core/src/test/java/acceptance/classification/UpdateClassificationAccTest.java:104]: No reason to set all this stuff just to see that the user is unauthorized.

File: lib/taskana-core/src/test/java/acceptance/classification/DeleteClassificationAccTest.java [hunk 6]
```diff
@@ -39,15 +39,6 @@ void testDeleteClassificationInDomain()
...
 
-  @WithAccessId(
-      user = "teamlead_1",
-      groups = {"group-1", "group-2"})
-  @Test
-  void testDeleteClassificationInDomainUserIsNotAuthorized() {
-    ThrowingCallable call = () -> classificationService.deleteClassification("L140101", "DOMAIN_A");
-    assertThatThrownBy(call).isInstanceOf(NotAuthorizedException.class);
-  }
-
```
[the remark this answers is not in the document]
Comment (holgerhagen) [reply lib/taskana-core/src/test/java/acceptance/classification/DeleteClassificationAccTest.java:50]: This was a duplicate in my opinion.

**Pull request merged**